Check updates returns a nil release when there is no one

Check updates returns a new release when there is no one. It causes a panic as describe bellow.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x721cca]

goroutine 1 [running]:
github.com/aureliano/caravela/updater.findUpdateUseCache({0xa68900, 0xc000109b30}, {0xa6cf30, 0xc00003a880})
	/go/pkg/mod/github.com/aureliano/caravela@v1.0.0/updater/find_update.go:46 +0x8a
github.com/aureliano/caravela/updater.FindUpdate({0xa68900?, 0xc000109b30?}, {0xa6cf30?, 0xc00003a880?}, {0xa667b8, 0xa}, 0x0?)
	/go/pkg/mod/github.com/aureliano/caravela@v1.0.0/updater/find_update.go:23 +0x51
github.com/aureliano/caravela.CheckUpdates({{0xa667b8, 0xa}, {0xa6cf30, 0xc00003a880}, 0x10e2100, 0x0})
	/go/pkg/mod/github.com/aureliano/caravela@v1.0.0/caravela.go:37 +0x136
```

--- a/updater/find_update.go
+++ b/updater/find_update.go
@@ -43,6 +43,10 @@ func findUpdateUseCache(client pvdr.HTTPClientPlugin, provider pvdr.UpdaterProvi
 			return nil, err
 		}
 
+		if release == nil {
+			release = &pvdr.Release{}
+		}
+
 		_ = provider.CacheRelease(*release)
 	}
 

